packages/common/i18n/resources.resjson

```diff
@@ -0,0 +1,5 @@
+{
+  "lib.common.localizer.account": "Account",
+  "lib.common.localizer.resourceGroup": "Resource Groups",
```

Comment:
Nit: Shouldn't this be a singular "Resource Group"?

Reply:
Fixed this, thanks! (It doesn't show up here in the diff due to rebasing).